src/wmtk/EdgeMeshOperationExecutor.hpp

```diff
@@ -20,7 +20,7 @@ class EdgeMesh::EdgeMeshOperationExecutor
     /**
      * @brief gather all simplices that are deleted in a split
      *
-     * The deleted simplices are exactly the open star of the edge
+     * The deleted simplex is the edge itself
```

Comment:
That helps but you should also state the structure of `std::array<std::vector<long>, 2>`. What is contained in that array and in the vectors.